pkgs/top-level/all-packages.nix

```diff
@@ -7581,6 +7581,8 @@ with pkgs;
 
   gdmap = callPackage ../tools/system/gdmap { };
 
+  gdtoolkit = callPackage ../development/tools/gdtoolkit { };
```

Comment:
suggestion: add gdtoolkit to `python-aliases.nix` so that people know that it got moved

Reply:
Sure, will do!

Reply:
fixed in rebased commits